client/src/components/FullBattlefield.js

```diff
@@ -32,17 +17,15 @@ class FullBattlefield extends Component {
           height: '100%',
         }}
       >
-        <Battlefield cards={cards} useStubs strongBorder />
-        <Battlefield cards={cards} useStubs strongBorder />
-        <Battlefield cards={cards} useStubs strongBorder />
-        <Battlefield cards={cards} useStubs strongBorder />
+        <Battlefield cards={cards} player={players[0]} isFullView strongBorder infoBoxPosition="right" />
```

Comment:
Shouldn't we map over the players array here?

Reply:
I just have it hard coded to the four battlefields since that is always the expected amount, and the infoBoxPosition are not exactly the same each time. And I think we want 4 battlefields no matter how many players there are (assuming we have a max of 4). Also this should have been players[0], then players[1], etc.